mobile/components/BookCard.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from "react";
+import { Card, Paragraph } from "react-native-paper";
+
+// @ts-ignore
+export function BookCard({ key, book }) {
```

Comment:
Would be best to use React.FC for such types, so that we get the `children` prop for free (even though we aren't using it)

```suggestion
interface Props {
  key: string;
  book: any;
}

export const BookCard: React.FC<Props> = ({ key, book }) => {
```